pkg/descheduler/node/node.go

```diff
@@ -70,35 +71,36 @@ func ReadyNodes(ctx context.Context, client clientset.Interface, nodeLister list
 
 	readyNodes := make([]*v1.Node, 0, len(nodes))
 	for _, node := range nodes {
-		if IsReady(node) {
+		if IsReady(ctx, node) {
 			readyNodes = append(readyNodes, node)
 		}
 	}
 	return readyNodes, nil
 }
 
 // IsReady checks if the descheduler could run against given node.
-func IsReady(node *v1.Node) bool {
+func IsReady(ctx context.Context, node *v1.Node) bool {
+	logger := klog.FromContext(ctx)
 	for i := range node.Status.Conditions {
 		cond := &node.Status.Conditions[i]
 		// We consider the node for scheduling only when its:
 		// - NodeReady condition status is ConditionTrue,
 		// - NodeOutOfDisk condition status is ConditionFalse,
 		// - NodeNetworkUnavailable condition status is ConditionFalse.
 		if cond.Type == v1.NodeReady && cond.Status != v1.ConditionTrue {
-			klog.V(1).InfoS("Ignoring node", "node", klog.KObj(node), "condition", cond.Type, "status", cond.Status)
+			logger.V(1).Info("Ignoring node", "node", klog.KObj(node), "condition", cond.Type, "status", cond.Status)
 			return false
 		} /*else if cond.Type == v1.NodeOutOfDisk && cond.Status != v1.ConditionFalse {
-			klog.V(4).InfoS("Ignoring node with condition status", "node", klog.KObj(node.Name), "condition", cond.Type, "status", cond.Status)
+			logger.V(4).Info("Ignoring node with condition status", "node", klog.KObj(node.Name), "condition", cond.Type, "status", cond.Status)
```

Comment:
E.g. how this log line will look different from the current one? The log line already has `node` in it. Is there more context that needs to be added here?